src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Controls/UnoWebView/UnoWebView.cs

```diff
@@ -133,7 +126,10 @@ private void OnNavigationStarting(WebView sender, WebViewNavigationStartingEvent
 					ClearCacheAndCookies();
 				}
 			}
-
+#if __IOS__
+			// Needed as webview keeps for unknown reason the token, delete cookies each time avoid this behavior
```

Comment:
If NSHttpCookieStorage delete works properly why do you need to add this to clear every time?

Reply:
Well I figured it out by testing ... but can't explain really why. Seems like if its not done here each time, somewhere on the disk it is saved and won't be deleted.

Reply:
I read [here](https://stackoverflow.com/questions/33156567/getting-all-cookies-from-wkwebview) that the WKWebview doesn't always save cookies in NSHttpCookieStorage **right away** so there's a chance it's being saved after the attempt to clear cookies.

Reply:
Well it seems like a big mess and that's why the behavior is different each time we test it if it is "unpredicatble" as stated in the SO post you linked. Maybe this could be the solution. 
But we need to find out how to change the process pool. 

> As long as each WKWebView share the same instance of WKProcessPool, those Session cookies will be passed back to the server for each request. If you change the process pool for a WKWebView, you are essentially removing the session cookies for all future requests